glossary/kmem-dax/_definition

```diff
@@ -0,0 +1,7 @@
+For a volatile use of persistent memory, there is an alternative for [Memory Mode](#memory-mode).
```

Comment:
KMEM DAX is a semi-transparent alternative to Memory Mode for volatile use of PMem.